config.js

```diff
@@ -0,0 +1,29 @@
+require('dotenv').config();
```

Comment:
That's right mother fuckers we are using multiple files here and being all modular so we don't end up with one big asss file